Applications/WebPortal/WebApp/library/source/pages/account_directory_page/create_account_page/create_account_page.tsx

```diff
@@ -324,8 +324,13 @@ export class CreateAccountPage extends React.Component<Properties, State> {
   }
 
   public componentDidMount(): void {
+    for(const country of this.props.countryDatabase) {
+      this.state.identity.country = country.code;
```

Comment:
Don't mutate ```state``` without also calling ```setState```.

What you need to do instead is move this for-loop into the promise, ie. into line 333.

Reply:
This is not yet resolved, only mark an issue as resolved once some change has been made to correct the issue. If you feel no change needs to be made or need clarification then reply to the issue indicating such.